src/dlgProfilePreferences.h

```diff
@@ -174,7 +175,7 @@ private slots:
 private:
     void setColors();
     void setColors2();
-    void setColor(QPushButton*, QColor&);
+    void setColor(QPushButton*, QColor&, bool allowAlpha);
```

Comment:
ℹ️ I think that if you define the default value here (rather than in the definition) it shows up in the "auto-completion" popup when you are editing in Qt Creator...